.circleci/config.yml

```diff
@@ -192,19 +193,20 @@ workflows:
                 - master
             tags:
               ignore: /^v.*/
-      # - deploy-to-k8s-prod:
-      #     name: deploy-to-k8s-prod
-      #     namespace: a2ml
-      #     release-name: a2ml
-      #     values-to-override: "image.tag=$CIRCLE_TAG"
-      #     context: eks
-      #     requires:
-      #       - docker-publish-tag
-      #     filters:
-      #       tags:
-      #         only: /^v.*/
-      #       branches:
-      #         ignore: /.*/
+      - deploy-to-k8s:
+          name: deploy-to-k8s
+          cluster-name: ${PROD_CLUSTER_NAME}
+          namespace: a2ml
+          release-name: a2ml
+          values-to-override: "image.tag=$CIRCLE_TAG"
+          context: eks
```

Comment:
The context will need to change, since there's a different AWS account & role that is assumed for production. I'm working on this now.